docs/guides/building/macos.md

```diff
@@ -19,10 +19,6 @@ GUI apps on macOS and Linux do not inherit the `$PATH` from your shell dotfiles
 The minimum version of the operating system required for a Tauri app to run in macOS is `10.13`. If you need support for newer macOS APIs like `window.print` that is only supported from macOS version `11.0`
```

Comment:
It isn't directly related to this change, but we do specify macOS 10.15 and above in the tauri readme (although that's more meant that we support 10.15 and above, not that it's a minimum)

Reply:
Ah interesting